dashboard/app/assets/stylesheets/application.css.scss

```diff
@@ -1606,18 +1606,23 @@ a.download-video {
   }
 }
 
+.level-group-number {
+  display: block;
+  font-size: 16px;
+  line-height: 24px;
+  float: left;
+  width: 30px;
+}
+
+.level-group-content {
+  overflow: hidden;
```

Comment:
I checked out the indentation and it works.  Thanks for doing that.  I might add additional indentation to get the textarea to line up with the multiple-choice answer indent, but can do that later.